lib/testLauncher/launcherLogger.js

```diff
@@ -102,6 +114,39 @@ function createTestPack() {
 	launcherLogger._(stColors.mild('Creating test pack run...'));
 }
 
+function createWriteStream(path, deviceId) {
+	const pathToFile = path + `/${deviceId}.log`;
+
+	if (fs.existsSync(pathToFile)) {
+		fs.unlinkSync(pathToFile);
+	}
+
+	return fs.createWriteStream(path + `/${deviceId}.log`, {flags: 'ax'});
+}
+
+function mkDirByPathSync(targetDir) {
+	const sep = path.sep;
+	const initDir = path.isAbsolute(targetDir) ? sep : '';
+	const baseDir = '.';
+
+	targetDir.split(sep).reduce((parentDir, childDir) => {
+		const curDir = path.resolve(baseDir, parentDir, childDir);
+
+		try {
+			fs.mkdirSync(curDir);
```

Comment:
'es6-promisify' is a good idea, not sure it would make sense to make it async in this case.

Reply:
@JeKachan I think in this case it's OK to use blocking operation. This is ran only on script start-up.

If we do convert it to async/await, we'll need to make sure it's wrapped to the promise chain. Furthermore, it must be the same promise as bootstrap process, otherwise we'll get race condition.